src/ocaml/utils/config.ml

```diff
@@ -32,83 +32,31 @@ let version = Sys.ocaml_version
 
 let flambda = false
 
-<<<<<<< HEAD
 let exec_magic_number = "Caml1999X550"
 let ext_obj = ".o_The boot compiler cannot process C objects"
 
-||||||| 78ff8bc3c0
-let exec_magic_number = "Caml1999X534"
-=======
-let exec_magic_number = "Caml1999X535"
->>>>>>> origin/main
     (* exec_magic_number is duplicated in runtime/caml/exec.h *)
 and cmi_magic_number =
   (* When bumping this number, be sure to also update ../typing/magic_numbers.ml *)
-<<<<<<< HEAD
   "Caml1999I550"
 and cmo_magic_number = "Caml1999O550"
 and cma_magic_number = "Caml1999A550"
-||||||| 78ff8bc3c0
-  "Caml1999I534"
-and cmo_magic_number = "Caml1999O534"
-and cma_magic_number = "Caml1999A534"
-=======
-  "Caml1999I535"
-and cmo_magic_number = "Caml1999O535"
-and cma_magic_number = "Caml1999A535"
->>>>>>> origin/main
 and cmx_magic_number =
   if flambda then
-<<<<<<< HEAD
-    "Caml1999y550"
-||||||| 78ff8bc3c0
-    "Caml2021y535"
-=======
-    "Caml2021y536"
->>>>>>> origin/main
+    "Caml2021y550"
   else
-<<<<<<< HEAD
-    "Caml1999Y550"
-||||||| 78ff8bc3c0
-    "Caml2021Y534"
-=======
-    "Caml2021Y535"
->>>>>>> origin/main
+    "Caml2021Y550"
 and cmxa_magic_number =
   if flambda then
-<<<<<<< HEAD
-    "Caml1999z550"
-||||||| 78ff8bc3c0
-    "Caml2021z535"
-=======
-    "Caml2021z536"
->>>>>>> origin/main
+    "Caml2021z550"
   else
-<<<<<<< HEAD
-    "Caml1999Z550"
+    "Caml2021Z550"
```

Comment:
I reverted this to the 1999 version. I'd like for this to get double-checked, because I'm not sure whether @ncik-roberts 's change was intentional, and the magic number system in flambda changed.

Reply:
It was unintentional. I think I found-and-replaced badly. There was a good deal of churn of magic numbers in the 5.2 merge, which we then reverted, so I’m not surprised I messed something up.